### ISSUE TITLE
registered logging issue

### ISSUE THREAD

**Issue by ConfusedVorlon** [CocoaLumberjack/CocoaLumberjack#102]
love CocoaLumberjack - thank you.

just tried to use registered logging, but I get errors with classes that don't implement the methods for isRegisteredClass

e.g.:

*** NSForwarding: warning: object 0x3aae0f1c of class '__NSMessageBuilder' does not implement doesNotRecognizeSelector: -- abort

and on the simulator:

checking class: NSXPCDecoder
2013-03-23 17:09:54.629 VLCStreamer[88695:c07] *** NSForwarding: warning: object 0x2530b48 of class 'Object' does not implement methodSignatureForSelector: -- trouble ahead
2013-03-23 17:09:54.629 VLCStreamer[88695:c07] *** NSForwarding: warning: object 0x2530b48 of class 'Object' does not implement doesNotRecognizeSelector: -- abort

Am I missing something?

thanks.

**Comment by ivanfemia** [comment]
I have this issue after I updated to the latest version (2.0.0) from cocoapod (0.36.3). Previously it was working fine (ver 2.0.0-beta4 `pod 'CocoaLumberjack', :git => "https://github.com/ValCapri/CocoaLumberjack.git"`).

My class is implementing the DDAbstractLogger.
I tried to import NSObject, but it is already imported by DDAbstractLogger, I tried to use protocol DDLogger, but no success.
I'm using swift

NSForwarding: warning: object 0x10013b0a0 of class does not implement methodSignatureForSelector: -- did you forget to declare the superclass of ?

NSForwarding: warning: object 0x10013b0a0 of class does not implement doesNotRecognizeSelector: -- abort
